python/ray/tune/tests/test_tune_save_restore.py

```diff
@@ -98,10 +98,12 @@ def _restore(self, exp_name, local_dir, absolute_local_dir):
             os.path.join(absolute_local_dir, exp_name)
         )
 
-        checkpoint_path = os.path.join(
-            local_dir, exp_name, trial_name, "checkpoint_000001/checkpoint.pkl"
+        checkpoint_dir = os.path.join(
+            local_dir, exp_name, trial_name, "checkpoint_000001"
```

Comment:
@justinvyu I changed the logic of the test here to pass in the directory instead of the file. Can you confirm this is the right way to do it?

Reply:
Also `test_tune_restore` seems to be [failing](https://buildkite.com/ray-project/oss-ci-build-pr/builds/33981#018a39a4-9c31-4eff-98fa-5278bab7af20) due to the same reason after this change, but I'm not sure exactly why. @krfricke's [PR](https://github.com/ray-project/ray/pull/38895) contains a fix for it.